oat-sa/browserslist-app-tao
# Dependencies update + Safari versions

Started out as an ISO27001-related update, but after that I went a bit further.

_Note: this repo doesn't use a `develop` branch, as it's so rarely touched._

### Changes
- dependencies update, up to latest version of all main deps
  `npm audit` before/after:
<img width="675" alt="Screenshot 2023-04-12 at 17 36 31" src="https://user-images.githubusercontent.com/43652944/231580758-7d4233bf-c61c-4977-940d-39d7fdda9021.png">
<img width="547" alt="Screenshot 2023-04-12 at 22 37 04" src="https://user-images.githubusercontent.com/43652944/231580791-ae747dda-e65d-4c02-8160-d8aaf52e13e0.png">

- improved the unit test (the app is not really well tested, but anyway...)
- refactored output for Safari, so it will only show the major versions

<img width="544" alt="Screenshot 2023-04-12 at 22 45 42" src="https://user-images.githubusercontent.com/43652944/231580705-2f6cd8d8-85fc-47ed-81aa-829ac323bda3.png">

### To test this
`npm ci`
`npx browserslist` - to see the raw data
`npm run test`
`npm run start` - serves the above site
